components/02-molecules/social-links/_social-links.scss

```diff
@@ -0,0 +1,15 @@
+@use '../../01-atoms/atoms';
+@use '../../00-tokens/tokens';
+
+.social-links__list {
+  @include atoms.list-reset;
+
+  display: flex;
```

Comment:
We need to make sure these wrap on smaller screens. Add `flex-wrap: wrap`

Reply:
Added flex-wrap: wrap.